### PR TITLE
bpo-42948: Document bytearray.copy()

### PR DESCRIPTION
**Pull request by terryjreedy**
https://bugs.python.org/issue42948

### CODE REVIEW

File: Misc/NEWS.d/next/Documentation/2021-01-23-12-23-34.bpo-42948.apqCwM.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Document bytearray.copy()
```
Comment (Fidget-Spinner) [inline Misc/NEWS.d/next/Documentation/2021-01-23-12-23-34.bpo-42948.apqCwM.rst]:
```suggestion
Document ``bytearray.copy()``
```

File: Doc/library/stdtypes.rst [hunk 2]
```diff
@@ -2636,6 +2636,15 @@ binary data. These restrictions are covered below.
    Using these ASCII based operations to manipulate binary data that is not
    stored in an ASCII based format may lead to data corruption.
 
+
+The following method is only for bytearray objects.
+
+.. method:: bytearray.copy()
+
+   Return a copy of the bytearray.  (Immutable bytes never need to be
+   copied.)
```
Comment (Fidget-Spinner) [inline Doc/library/stdtypes.rst]: Sorry if I'm being a pedant here. On first read I didn't immediately understand what the parenthesized sentence meant. After more thought, I'm guessing you were referring to the bytes object?

```suggestion
   Return a copy of the bytearray.  (Bytes objects are immutable and never need
   to be copied.)
```

**Review by Fidget-Spinner — APPROVED** [review]
This reads quite nicely, I like it!.

File: Misc/NEWS.d/next/Documentation/2021-01-23-12-23-34.bpo-42948.apqCwM.rst (hunk 1 above)
Comment (terryjreedy) [reply Misc/NEWS.d/next/Documentation/2021-01-23-12-23-34.bpo-42948.apqCwM.rst]: I am trusting that you are correct and that linking from blurbs works.  This also need a period.

File: Doc/library/stdtypes.rst (hunk 2 above)
Comment (terryjreedy) [reply Doc/library/stdtypes.rst]: Thank you.  The doc should be clear to people who are not 20+ year veterans of Python ;-).  I left this open after CI passed to allow for random review.  I wish more newer contributors should do them.

**Comment by terryjreedy** [comment]
Closing because .copy is one of 8 mutable sequence methods, which are documented elsewhere.

**Pull request closed without merging**